app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
- [ ] In your comments controller there should be method called `new` as you have created in posts controller. Kindly add and resubmit it again. After completing these requirements please make sure it's working as expected.

Reply:
Well noted. Thank you🙏